### ISSUE TITLE
Questions about model training

### ISSUE THREAD

**Issue by jianingwangind** [HuangJunJie2017/BEVDet#3]
Great work!
Regarding to model training, I tried to use CenterPoint head for 3D detection in BEV space  with image inputs. The training loss looks fine to me, however, the resulting AP is 0. Also, I have tried your training configurations, such as the optimizer type and schedules, the outcoming AP is still near 0, making nonsense.
It would be great if you can share me with some insights.

**Comment by HuangJunJie2017** [comment]
The code has been released. Hope this can help solving your problem. Thanks for your attention!
